src/main/java/org/myrobotlab/opencv/ImageFileFrameGrabber.java

```diff
@@ -79,6 +79,13 @@ public Frame grab() {
     // increment our count.
     grabCount++;
     grabCount = grabCount % imageFiles.size();
+    if (converter == null) {

```

Comment:
converter should never be null.   why the need for the check here?

Reply:
npe's in the unit tests